toolchain/sem_ir/file.h

```diff
@@ -34,6 +35,12 @@ namespace Carbon::SemIR {
 // Provides semantic analysis on a Parse::Tree.
 class File : public Printable<File> {
  public:
+  // Used to return information about an integer type in `GetIntTypeInfo`.
+  struct IntTypeInfo {
+    bool is_signed;
+    IntId bit_width = IntId::Invalid;
```

Comment:
```suggestion
    IntId bit_width;
```

Looks like this default is unused, suggesting removal.